Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -675,6 +688,16 @@ theorem map_le_iff_le_comap {G' : SimpleGraph W} (f : G →g G') (H : G.Subgraph
     rintro u u' hu rfl rfl
     exact (h.2 hu).2
 
+open scoped Classical in
+noncomputable instance [Finite V] : Fintype G.Subgraph := by
+  have := (nonempty_fintype V).some
```

Comment:
Let's also have the function Fintype V -> Fintype G.Subgraph

Reply:
Sorry, I am not sure what you want here. Isn't this instance maximally general already? Note that I am producing data out of nothing (`Finite -> Fintype`) because there is no way the `Fintype G.Subgraph` computable